lib/lte_link_control/modules/xsystemmode/Kconfig

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config LTE_LC_XSYSTEMMODE
+	bool "Enable LTE XT3412 functionalities"
+	default y
+	depends on !LTE_NETWORK_MODE_DEFAULT
```

Comment:
MOSH doesn't compile with this dependency. Not sure what the intention of this is.

Reply:
I was experimenting, targeting `https_client` sample.